src/TeamStyled.js

```diff
@@ -1,5 +1,7 @@
 import styled from 'styled-components';
 
+
+{/***********************************
```

Comment:
Codacy found an issue: [Block is redundant.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6058781)